api/WebSocket.json

```
@@ -597,7 +602,7 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": "12"
```

Again, doing some research earlier, this appears to have been supported much earlier:

```suggestion
              "version_added": "6"
```